lua/telescope/builtin/internal.lua

```diff
@@ -1,12 +1,15 @@
 local actions = require "telescope.actions"
 local action_set = require "telescope.actions.set"
 local action_state = require "telescope.actions.state"
+local config = require "telescope.config"
```

Comment:
Not needed we already have `conf` imported (unless you need something else from config, doesn't look like it rn)

Reply:
Yes, slight oversight sorry :) fixed locally and to be updated once more comes in